core/src/main/scala/sttp/tapir/model/ServerResponse.scala

```diff
@@ -4,7 +4,7 @@ import sttp.model.{Header, Headers, ResponseMetadata, StatusCode}
 
 import scala.collection.immutable.Seq
 
-case class ServerResponse[B](code: StatusCode, headers: Seq[Header], body: Option[B]) extends ResponseMetadata {
+case class ServerResponse[+B](code: StatusCode, headers: Seq[Header], body: Option[B]) extends ResponseMetadata {
```

Comment:
Ended up doing this to fix the compilation errors on `http4sServer3` and `vertxServer3` projects, but not sure if it's correct.

Reply:
Hm what was the compilation error? :) I suppose the variance here is fine, but it's interesting that it only is a problem in the scala3 versions